ui/app/templates/rule_field.html

```diff
@@ -0,0 +1,19 @@
+<h5 ng-show="(rule[fieldname] !== null && rule[fieldname] !== undefined) || (showsc && rule.scheduled_change[fieldname] !== null && rule.scheduled_change[fieldname] !== undefined)">
+    {{ fieldtitle }}
+    <span ng-show="help && (rule[fieldname] !== null || (showsc && rule.scheduled_change[fieldname] !== null))"
+          class="glyphicon glyphicon-question-sign" title="{{ help }}"></span>
+    :
+<b>
+<!-- we have two display modes: with and without scheduled changes.
+     if scheduled changes are being shown, we shown any fields that are
+     changing as a "diff", where the current value is struck out, and
+     the scheduled change version is in green. if scheduled changes aren't
+     being shown, we just show the current value plainly -->
+<span ng-style="!rule.rule_id && (showsc && rule.scheduled_change !== null && rule[fieldname] !== rule.scheduled_change[fieldname]) ? {'text-decoration': 'line-through', 'color': 'red'} : {};">
```

Comment:
I'm surprised rulefields aren't struck out, like @nthomas-mozilla reported. Could it be because of `rule.rule_id` or showsc being truthy/falsey?

Reply:
It turns out that this was because the "rule_id" we should be making this decision on is the one that comes from the URL (that let's us know "are we on the history page or not") - not the one that's part of each individual rule object.